mix_helpers.exs

```diff
@@ -9,6 +9,12 @@ defmodule Mix.Appsignal.Helper do
 
   @max_retries 5
 
+  @proxy_env_vars [
+    "APPSIGNAL_PROXY",
```

Comment:
My preference would be to call this `APPSIGNAL_HTTPS_PROXY` make it as much in line with the system ENV as possible. Your thoughts @jeffkreeftmeijer?

Reply:
We should use `APPSIGNAL_HTTP_PROXY` since that's already a config option: https://docs.appsignal.com/elixir/configuration/options.html#option-http_proxy
No need to add a new one.